init/init.go

```diff
@@ -168,10 +171,21 @@ func getLaunchConfig(cfg *config.CloudConfig, dockerCfg *config.DockerConfig) (*
 	return &launchConfig, args
 }
 
+func isInitrd() bool {
+	var stat syscall.Statfs_t
+	syscall.Statfs("/", &stat)
+	return stat.Type == 0x01021994
```

Comment:
I don't know for sure what to do here, but some conversation is probably necessary. 
In #559 it was fixed by first casting the constant `syscall.LINUX_REBOOT_CMD_POWER_OFF` to `uint` and then back to `int`.